mountinfo/go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/moby/sys/mountinfo
 
 go 1.16
 
-require golang.org/x/sys v0.0.0-20220412211240-33da011f77ad
+require golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
```

Comment:
Probably good to update this dependency in all the modules in this repository (`mount`, `mountinfo`, `symlink`, `signal`)

diff is: https://github.com/golang/sys/compare/33da011f77ad...bc2c85ada10a

Reply:
Sure, this is a good idea.